### ISSUE TITLE
[BUG] Negative RGB values for country colours appear garbled

### ISSUE THREAD

**Issue by Altarya** [herbix/hoi4modutilities#44]
**Describe the bug**
Countries with negative RGB values appear garbled.

**To Reproduce**
Set a country's rgb values to be negative

**Expected behavior**
Hoi4 lets you set countries' rgb values to be negative(max -256) so they act as offsets to the engine's post processing, i supposed the expected behaviour here would be for those values to default 0 within the preview.

**Screenshots**
![image](https://user-images.githubusercontent.com/20309104/146104070-9a5ec7b6-1db2-49f4-bbdf-7f830a555c60.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - VSCode version: 1.63 
 - Extension version: 0.5.0

**Console Logs**
Not needed.

**Comment by herbix** [comment]
I didn't know that country color can have negative values.
Can you share me an article that describe how negative color values works?

**Comment by Altarya** [comment]
> 

I dont think there is a full article with an explanation(I picked this up from other modders) but what they do is they offset the colours the game would have rendered from post processing

For example:
You have a country with the colour 255,0,0
However ingame due to post processing it shows up as 255,43,60
Lets say you want that country to show up as 255,0,0 what you would would be to assign its colour code as 255,-43,-60 and thus offsetting the post processing of the game(tho its not universal since post processing depends on your camera distance, position, time of ingame day, and so on).

Since you only use this feature to achieve specific, generally vibrant, colours where one of your rgbs would be 0 its a thing that's rarely used. The solution like I suggested would be to just add a simple check if the rgb values are negative and if yes then set them to 0 for the display.

**Comment by Altarya** [comment]
Let me explain with some screenshots.
Here the tag has the rgb value 255,0,0
![image](https://user-images.githubusercontent.com/20309104/146112314-f2dbf900-25ce-4692-8b3e-a29ef3afd986.png)

and here with it set to 255.-255,-255
![image](https://user-images.githubusercontent.com/20309104/146112423-1a8eefa7-29c2-4fdf-90a4-2a94f7f4f235.png)

As you can see in the first image the post processing added some colouring to the green and blue channels

**Comment by Altarya** [comment]
Yup, that works, the preview colours were already always gooing to be a little off due to post processing anyhow, but still serve their purpose as a general indication.

Thanks!
